package.json

```diff
@@ -384,7 +384,7 @@
     "enzyme": "^3.11.0",
     "enzyme-adapter-react-16": "^1.15.2",
     "eslint-config-mongodb-js": "^5.0.3",
-    "hadron-build": "^23.2.6",
+    "hadron-build": "^23.3.0",
```

Comment:
`23.3.1` :)

Reply:
I’ve updated to that – I have no strong feelings about whether we want to drop the write-to-file step then, after sleeping about it it seems more likely to me that somebody would accidentally print environment variables like we did before than accidentally publish files from `/tmp/`, but obviously either way works just fine for me.